setup.cfg

```diff
@@ -0,0 +1,30 @@
+[flake8]
+# max line length for black
+max-line-length = 88
+target-version = ['py37']
```

Comment:
I think we can already target 38 or 39 :)

Reply:
Also nowadays I would tend to use ruff instead of flake8 but not big deal, especially on a small code base.

Reply:
I mindlessly copied files that we copied from sklearn at the time when sklearn-numba-dpex was created but it's been evolving since then, we should indeed update and apply ruff on all our repos now...